ownca/utils.py

```diff
@@ -53,12 +65,14 @@ def file_data_status(ca_status):
         return None
 
 
-def _create_ownca_dir(ownca_dir):
+def _create_ownca_dir(ownca_dir: str) -> None:
     """
     Creates the CA directory.
 
-    :param ownca_dir: :string: full path directory for ownca
+    :param ownca_dir: full path directory for ownca
+    :type ownca_dir: string, required
     :return: bool
```

Comment:
The return type in the function declaration (None) doesn't match the return type in the docstring (bool). It appears that this function has no return statement, so we should change the docstring.

Reply:
Nice catch